esx_service/vmdk_ops.py

```diff
@@ -1757,6 +1791,10 @@ def execRequestThread(client_socket, cartel, request):
         reply_string = err("Server returned an error: {0}".format(repr(ex_thr)))
         send_vmci_reply(client_socket, reply_string)
 
+    # Decrease the in-flight ops counter
+    opsCounter.decr()
+    logging.debug("execRequestThread: opsCounter=%d", opsCounter.value)
```

Comment:
the value can change between lines 1795 and 796 if there is any other thread touching it

Reply:
Will fix.